exampleCourse/questions/addBinaryStringElement/info.json

```diff
@@ -0,0 +1,7 @@
+{
+    "uuid": "0387c7f5-0c36-447c-9739-e6794d234a6b",
+    "title": "Add binary numbers",
```

Comment:
I suggest we not bother with this question, because the other question already shows how to use the the element really clearly and I'd rather keep the number of questions under control :-)